ui/apps/platform/src/Containers/Policies/Wizard/PolicyWizard.tsx

```diff
@@ -149,73 +150,105 @@ function PolicyWizard({ pageAction, policy }: PolicyWizardProps): ReactElement {
             >
                 <FormikProvider value={formik}>
                     <Wizard
+                        title={`${pageAction} policy wizard`}
                         navAriaLabel={`${pageAction} policy steps`}
-                        mainAriaLabel={`${pageAction} policy content`}
                         onClose={closeWizard}
                         onSave={submitForm}
-                        hasNoBodyPadding
-                        steps={[
-                            {
-                                id: 1,
-                                name: 'Policy details',
-                                component: (
+                        isVisitRequired={!canJumpToAny}
+                        onStepChange={onStepChange}
+                    >
+                        <WizardStep
+                            name="Policy definition"
+                            id="policy-definition"
```

Comment:
Asking, not suggesting change, about pro and con to define as a constant (only occurrence is here).